### PR TITLE
Route local Claude events through Sketch agent

### PR DESCRIPTION
**Pull request by rnijhara**
## What changed

- Added hidden `<local_claude_session_event>` context so Claude Code hook events wake the Sketch agent without being treated as user messages.
- Persisted local Claude session origin runtime metadata: workspace, queue key, conversation/thread ids, agent instructions, and allowed tools.
- Replaced direct Slack/WhatsApp event notification with an async local Claude event dispatcher that reruns the Sketch agent in the originating context and sends only the agent's final useful response back to the user-visible channel.
- Added coverage for prompt rendering, event route dispatch, session event recording, and migration ordering.

## Why

Claude Code hook events are internal coordination signals. They should not appear directly in chat, but they should still let Sketch inspect the local session and respond naturally to the user when there is a useful outcome or a clarification needed.

## Validation

- `pnpm biome check .`
- `pnpm typecheck`
- `pnpm test`
- `pnpm build`
- Pre-push hook reran lint, typecheck, tests, and build successfully.

### CODE REVIEW

File: packages/server/src/local-devices/claude-event-dispatcher.ts [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        payload,
+      },
+    });
+    const integrationMcpServers = deps.buildMcpServers ? await deps.buildMcpServers(user?.email ?? null) : {};
+    const result = await deps.runAgent({
+      db: deps.db,
+      workspaceKey: session.origin_workspace_key,
+      userMessage,
+      workspaceDir,
+      claudeConfigDir: deps.config.CLAUDE_CONFIG_DIR,
```
[the remark this answers is not in the document]
Comment (rnijhara) [reply packages/server/src/local-devices/claude-event-dispatcher.ts]: Fixed in `cfa5718`.

The local Claude session now persists `origin_org_context_enabled` from the originating agent run, based on whether `claudeConfigDir` was present. Event replay uses that stored value for both the hidden `orgDir` context and the replay `claudeConfigDir`, so WhatsApp fallback-agent sessions continue without org context. I also added dispatcher tests for disabled fallback-origin replay and legacy null/default-enabled replay.
